pkg/etw/fieldopt.go

```diff
@@ -55,6 +55,14 @@ func StringField(name string, value string) FieldOpt {
 	}
 }
 
+// UnicodeStringField adds a single UTF-16 string field to the event.
+func UnicodeStringField(name string, value string) FieldOpt {
+	return func(em *eventMetadata, ed *eventData) {
+		em.writeField(name, inTypeUnicodeString, outTypeString, 0)
```

Comment:
Same question here about the out type.

Reply:
Sorry for long delay, I missed the notification.

I followed the manifest here, which states
`<data name="Message" inType="win:UnicodeString" outType="xs:string"/>`

Unfortunately I don't have enough ETW technical background to know if `outTypeDefault` would suffice. The most I can do is to test if it works with `outTypeDefault`, but I can't be sure about some corner cases.